File: test/AdventureWorksLT/UdtFlagAttribute.cs
using System;
using DevZest.Data;
using DevZest.Data.Annotations.Primitives;

namespace DevZest.Samples.AdventureWorksLT
{
    public sealed class UdtFlagAttribute : ColumnAttribute
    {
        protected override void Initialize(Column column)
        {
            column.Nullable(false);
        }
    }
}

File: test/AdventureWorksLT.VB/UdtFlagAttribute.vb
Imports DevZest.Data
Imports DevZest.Data.Annotations.Primitives

Namespace DevZest.Samples.AdventureWorksLT
    Public NotInheritable Class UdtFlagAttribute
        Inherits ColumnAttribute

        Protected Overrides Sub Initialize(column As Column)
            column.Nullable(False)
        End Sub
    End Class
End Namespace
